### PR TITLE
Make use of launcher wheel

### PR DESCRIPTION
**Pull request by kalikiana**
Fixes: https://progress.opensuse.org/issues/108530

### CODE REVIEW

**Comment by okurz** [comment]
You might be able to test it (maybe only on o3-openqaworker7) by forking
https://build.opensuse.org/package/show/devel:openQA/isotovideo-qemu-kvm
and building your own container from https://github.com/kalikiana/os-autoinst/tree/git_wheel_support and then

```
openqa-clone-job --within-instance https://openqa.opensuse.org/t... WORKER_CLASS=openqaworker7 \
ISOTOVIDEO="podman run --rm -it registry.opensuse.org/home/kalikiana/branches/devel/openqa/containers/isotovideo:qemu-kvm /usr/bin/isotovideo -d" \
CASEDIR=https://github.com/kalikiana/os-autoinst-distri-openQA/tree/wheel_launcher
```

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
https://openqa.opensuse.org/tests/2494128

File: wheels.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+wheels:
```
Comment (okurz) [inline wheels.yaml:1]:
```suggestion
version: v0.1
wheels:
```

**Comment by okurz** [comment]
o3 workers were outdated, see https://progress.opensuse.org/issues/114941 . Now retriggered as https://openqa.opensuse.org/tests/2494138

**Review by okurz — CHANGES_REQUESTED** [review]
https://openqa.opensuse.org/tests/2494766#step/dashboard/5 fails with

```
# Test died: Odd name/value argument for subroutine 'OpenQA::Wheel::Launcher::start_gui_program' at /var/lib/openqa/pool/14/os-autoinst-distri-openQA//tests/openQA/dashboard.pm line 11.
```

File: wheels.yaml (hunk 1 above)
Comment (kalikiana) [reply wheels.yaml:2]: I don't see a difference

**Comment by okurz** [comment]
Retriggered test https://openqa.opensuse.org/tests/2494826

**Review by okurz — APPROVED** [review]
https://openqa.opensuse.org/tests/2494826#step/dashboard/7 is good

File: wheels.yaml (hunk 1 above)
Comment (okurz) [reply wheels.yaml:2]: Because you meanwhile added it

**Pull request merged**